setup.sh

```diff
@@ -105,7 +105,7 @@ printf "\n${CYAN}Cloudant started.${PLAIN}\n"
 
 ## create database		
 printf "\n${RED}>> Creating database in Cloudant${PLAIN}"		
-curl --request PUT --url http://$USER:$PASSWORD@$HOST:$PORT/$DATABASE > /dev/null 2>&1		
+curl -i -X PUT http://$USER:$PASSWORD@$HOST:$PORT/$DATABASE > /dev/null 2>&1		
```

Comment:
Can we remove `2>&1`, so that when the request fails, the errors are printed by the `setup.sh` script?

I also believe that `-i` is not necessary, since you are discarding the output to `/dev/null` anyways.

Quoting from `man curl`:

> -i, --include
> Include the HTTP-header in the output. The HTTP-header includes things like server-name, date of the document, HTTP-version and more...
>
> See also -v, --verbose.

Proposed line:


```suggestion
curl -X PUT http://$USER:$PASSWORD@$HOST:$PORT/$DATABASE > /dev/null	
```

Reply:
npm run mocha
npm run test 

fail with some errors on my laptop, but works in CI build?

Reply:
hmm some tests fail on laptop with master branch too.

Why docker-compose use the `:latest` tag, but `test.js` use `:2.0.1` ?

Reply:
seem to be the same

![image](https://user-images.githubusercontent.com/6864736/67108242-1588f480-f19c-11e9-989c-37ca9a976f67.png)